src/main/java/com/pileproject/drivecommand/machine/device/input/simulation/SimulatedColorSensor.java

```diff
@@ -0,0 +1,38 @@
+package com.pileproject.drivecommand.machine.device.input.simulation;
```

Comment:
Anyway, I agree with @amiq11's idea in the case we use these classes just for testing because users can easily understand where these mocks are placed. 
Let's regard the `mock` package as a small repository for mocks (It may help some developers who implement a communication for other robots).

If some classes, especially `*Controller`s, in this PR are general enough (i.e., it can be used not only for such testing/mocking but also for other purposes), we shouldn't them into the `mock` package.